app/views/ReportView/components/MutationSignatures/index.jsx

```diff
@@ -161,8 +169,8 @@ const MutationSignatures = () => {
           />
         </>
       ) : (
-        <LinearProgress />
-      )}
+          <LinearProgress />
+        )}
```

Comment:
Indentation was added here resulting in a linting error

Reply:
weird, that was auto-fixed by the linter... do we have different settings somehow? will compare offline

Reply:
note: we determined that something odd is going on where running eslint via command line doesn't show this error but sometimes it does in vscode. @Anna-log7  is currently strengthening the linting rules in another branch which should hopefully resolve this